apps/frontend/src/components/form/inputTypes/FormSelect.tsx

```diff
@@ -6,16 +6,37 @@ interface FormSelectProps {
   selectList: string[][];
   id: string;
   field: FieldInputProps<any>;
+  onChange?: (value: string) => void;
+  isDisabled?: boolean;
 }
 
 const FormSelect: React.FC<FormSelectProps> = ({
   placeholder,
   selectList,
   id,
   field,
+  onChange,
+  isDisabled,
 }) => {
+
+  const handleChange = (value: any) => {
```

Comment:
Why is it necessary to do this twice (once here and also in the FormInput component)?